deployments/puppet/README.md

```diff
@@ -0,0 +1,65 @@
+# Splunk OpenTelemetry Connector Puppet Module
+
+This is a Puppet module that will install and configure the Splunk
+OpenTelemetry Connector.
+
+Currently, the following Linux distributions and versions are supported:
+
+- Amazon Linux: 2
+- CentOS / Red Hat / Oracle: 7, 8
+- Debian: 8, 9, 10
+- Ubuntu: 16.04, 18.04, 20.04
+
+> Note: `systemd` is required to be installed on the host for service
+> management.
+
+To use this module, include the `splunk_otel_collector` class in your
+manifests.  For example, the simplest deployment definition with the
+default parameters would be (replace `SPLUNK_ACCESS_TOKEN` with your Splunk
+access token to authenticate requests and `SPLUNK_REALM` for the realm to send
+the data to):
+
+```ruby
+class { splunk_otel_collector:
+  splunk_access_token => 'SPLUNK_ACCESS_TOKEN',
+  splunk_realm => 'SPLUNK_REALM',
+}
+```
+
+This class accepts the following parameters:
+
+| Name | Description | Default value |
+| :--- | :---------- | :------------ |
+| `splunk_access_token` | **Required**: The Splunk access token to authenticate requests. | None |
+| `splunk_realm` | Which realm to send the data to.  The Splunk ingest and API URLs will be inferred by this value.  The `SPLUNK_REALM` environment variable will be set with this value for the collector service. | `us0` |
```

Comment:
I think @jrcamp had the request in the ansible role to make this required w/o default